metricbeat/etc/fields.yml

```diff
@@ -8,7 +8,28 @@ defaults:
   ignore_above: 1024
 
 sections:
+  - ["general", "General event data"]
   - ["redis-info", "Redis Info Stats"]
+
+general:
+  type: group
+  description: >
+    General fields for each event
+  fields:
+    - name: response_time
```

Comment:
The name sounds ok to me either way. I'm partial to response_time but don't let me opinion stop you :)

As for nanoseconds... I think we should not use them at all right now. It looks like Elasticsearch and Kibana both do not support nanoseconds (https://github.com/elastic/kibana/issues/2498 and https://github.com/elastic/elasticsearch/issues/10005). We should double check but if that's true, I think nanoseconds wouldn't be good right now.

As for the use cases, I have worked with people in past lives who needed nanosecond precision on service/server/app response times. That sounds nuts (and it is) but consider that they were working with the stock market :)